backend/endpoints/rooms.py

```diff
@@ -38,55 +38,31 @@ async def create_room(
     Raises
         HTTP_424_FAILED_DEPENDENCY: room creation unsuccessful
     """
-    room_data = request.dict()
-    plugin_name = room_data.get("plugin_name")
-    room_name = room_data.get("room_name")
-    room_member_ids = list(room_data.get("room_members").keys())
-    rooms = await get_org_rooms(org_id=org_id, plugin=plugin_name)
 
-    if rooms is not None:
-        if plugin_name == Plugin.CHANNEL and room_name.casefold() in [
-            room["room_name"].casefold() for room in rooms
-        ]:
-            raise HTTPException(
-                status_code=status.HTTP_200_OK, detail={"room_name": room_name}
-            )
+    room_obj = Room(**request.dict(), org_id=org_id, created_by=member_id)
+    response = await DB.write("rooms", data=room_obj.dict())
```

Comment:
How about this:
```suggestion
    response = await DB.write(ROOM_COLLECTION, data=room_obj.dict())
```

Reply:
Fixed

Reply:
Thank you. The constant is not used in `db.py,` so it might make sense to move it to `rooms.py`.
What do you think about it? @o4codes